rllib/utils/replay_buffers/episode_replay_buffer.py

```diff
@@ -487,12 +494,15 @@ def _sample_episodes(
 
         # Use our default values if no sizes/lengths provided.
         batch_size_B = batch_size_B or self.batch_size_B
-        # TODO (simon): Implement trajectory sampling for RNNs.
-        batch_length_T = batch_length_T or self.batch_length_T
 
-        # Sample the n-step if necessary.
-        actual_n_step = n_step or 1
-        random_n_step = isinstance(n_step, tuple)
+        # If no sequence should be sampled, we sample n-steps.
```

Comment:
Can we add an error/assertion that makes sure users never combine n-step with sequence sampling?

Reply:
Will do. Great idea!